src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -16,8 +16,11 @@
             function(xhr, el) {
 
                 /* Evaluate embedded scripts inside the toolbar */
-                var rx=/<script>(.*?)<\/script>/g, s;
-                while (s = rx.exec(el.innerHTML)) {eval(s[1]);};
+                var i, scripts = [].slice.call(el.getElementsByTagName('SCRIPT'));
```

Comment:
why not using `el.querySelectorAll('script')` as in the other parts of the script?